entity-framework/efcore-and-ef6/index.md

```diff
@@ -82,79 +79,76 @@ The EF Core column indicates the product version in which the feature first appe
 
-| **Feature**                                           | **EF6**            | **EF Core**                           |
-|:------------------------------------------------------|:-------------------|:--------------------------------------|
-| .NET Framework                                        | Yes                | 1.0 (removed in 3.0)                  |
-| .NET Core                                             | Yes (added in 6.3) | 1.0                                   |
-| Mono & Xamarin                                        |                    | 1.0 (in-progress)                     |
-| UWP                                                   |                    | 1.0 (in-progress)                     |
+<sup>3</sup> EF Core database providers implemented by third-parties may be delayed in updating to new major versions of EF Core. See [Database Providers](../core/providers/index.md) for more information.
 
-## Guidance for new applications
+<sup>4</sup> The SQL Server Compact and Jet providers only work on .NET Framework (not on .NET Core).
 
-Consider using EF Core for a new application if both of the following conditions are true:
-* The app needs the capabilities of .NET Core. For more information, see [Choosing between .NET Core and .NET Framework for server apps](https://docs.microsoft.com/dotnet/standard/choosing-core-framework-server).
-* EF Core supports all of the features that the app requires. If a desired feature is missing, check the [EF Core Roadmap](xref:core/what-is-new/index) to find out if there are plans to support it in the future. 
+### Supported platforms
 
-Consider using EF6 if both of the following conditions are true:
-* The app will run on Windows and the .NET Framework 4.0 or later.
-* EF6 supports all of the features that the app requires.
+EF Core 3.1 runs on .NET Core and .NET Framework, through the use of .NET Standard 2.0. However, EF Core 5.0 will not run on .NET Framework. See [Platforms](../core/platforms/index.md) for more details.
 
-## Guidance for existing EF6 applications
+EF6.4 runs on .NET Core and .NET Framework, through multi-targeting.
 
-Because of the fundamental changes in EF Core, we do not recommend moving an EF6 application to EF Core unless there is a compelling reason to make the change. If you want to move to EF Core to use new features, make sure you're aware of its limitations. For more information, see [Porting from EF6 to EF Core](porting/index.md). **The move from EF6 to EF Core is more a port than an upgrade.**
+## Guidance for new applications
+
+Use EF Core on .NET Core for all new applications unless the app needs something that is [only supported on .NET Framework](https://docs.microsoft.com/dotnet/standard/choosing-core-framework-server).
+
+## Guidance for existing EF6 applications
```

Comment:
Link to the [porting guide](https://docs.microsoft.com/en-us/ef/efcore-and-ef6/porting/)?

Reply:
Good point. Although the porting guide is also something that needs updating!